signac/common/config.py

```diff
@@ -52,8 +52,8 @@ def _raise_if_older_schema(root):
         raise IncompatibleSchemaVersion(
             "The signac schema version used by this project is "
             f"{schema_version}, but signac {__version__} requires "
-            f"schema version {SCHEMA_VERSION}. Try running python -m "
-            "signac migrate."
+            f"schema version {SCHEMA_VERSION}. Try running 'python -m "
+            "signac migrate'."
```

Comment:
I'm going to undo this change because it implies a consistency in our formatting of this stuff that isn't there right now (for example [here's one message](https://github.com/glotzerlab/signac/blob/master/signac/__main__.py#L1285), and [here's another](https://github.com/glotzerlab/signac/blob/master/signac/contrib/project.py#L849)). I'm fine with standardizing the code snippets in our messages across the code base, but we should do so consistently in a separate PR.

Reply:
I just wanted to match this one, in the aim of standardization: https://github.com/glotzerlab/signac/blob/4079d0b1e1d5a85f0b3f54f95fe600eaf9838fc7/signac/contrib/project.py#L235

These two are effectively the same message, so it's weird to write the same message in two different ways even if the rest of the codebase has other messages that are styled differently.

Reply:
@vyasr Would you be willing to make a PR with that standardization? The change you reverted was my attempt at moving in that direction. 🙃